sklearn/preprocessing/mice.py

```diff
@@ -0,0 +1,365 @@
+# Authors: Sergey Feldman <sergeyfeldman@gmail.com>
+# License: BSD 3 clause
+
+from copy import deepcopy
+from time import time
+
+import numpy as np
+
+from .imputation import _get_mask, Imputer
+from ..base import BaseEstimator, TransformerMixin
+from ..dummy import DummyRegressor
+from ..preprocessing import normalize
+from ..utils import check_array
+from ..utils.validation import check_is_fitted
+
+__all__ = [
+    'MICEImputer',
+]
+
+
+class MICEImputer(BaseEstimator, TransformerMixin):
+    """
+    Basic implementation of MICE package from R.
+    This version assumes all of the columns are Gaussian.
+
+    Parameters
+    ----------
+    missing_values : integer or "NaN", optional (default="NaN")
+        The placeholder for the missing values. All occurrences of
+        missing_values will be imputed. For missing values encoded as
+        np.nan, use the string value "NaN".
+
+    imputation_order : str, optional, default="monotone"
```

Comment:
The convention will give: `imputation_order : str, optional (default="monotome")